task/06-conditions-n-loops-tasks.js

```diff
@@ -119,7 +150,11 @@ function isTriangle(a,b,c) {
  *  
  */
 function doRectanglesOverlap(rect1, rect2) {
-    throw new Error('Not implemented');
+    if (rect1.top + rect1.height > rect2.top && rect1.left + rect1.width > rect2.left) {
```

Comment:
use return <bolean expression> instead of if(<bolean expression>) return true else return false